bin/dev-ssl

```diff
@@ -0,0 +1,94 @@
+#!/usr/bin/env bash
+set -eo pipefail
```

Comment:
Have you run `shellcheck` on this script?

Reply:
Yep 😁